invenio/templates/configurations/logstash.yaml

```diff
@@ -53,66 +53,125 @@ data:
+      if [@metadata][ua_name] {
+          mutate {
+              add_field => { "[data][ua/family]" => "%{[@metadata][ua_name]}" }
+          }
+      }
+      # add ua/patch only if set
+      if [@metadata][ua_patch] {
+          mutate {
+              add_field => { "[data][ua/full]" => "%{[@metadata][ua_name]}/%{[@metadata][ua_major]}.%{[@metadata][ua_minor]}.%{[@metadata][ua_patch]}" }
+          }
+      } else {
+          mutate {
+              add_field => { "[data][ua/full]" => "%{[@metadata][ua_name]}/%{[@metadata][ua_major]}.%{[@metadata][ua_minor]}" }
+          }
+      }
+      # add ua/os only if set
+      if [@metadata][ua_os] {
+          mutate {
+              add_field => { "[data][ua/os]" => "%{[@metadata][ua_os]}" }
+          }
+      }
+      # add a field with the category, based on the url
+      grok {
+         match => { "url" => ["^/(api/)?record/%{INT}/files/", "^/api/files/%{UUID}/", "^/api/deposit/depositions/%{INT}/files"]}
+         add_field => {"[data][request_category]" => "files" }
+      }
+      if "_grokparsefailure" in [tags] {
+        grok {
+           match => { "url" => "^/%{WORD:[data][request_category]}"}
+           remove_tag => "_grokparsefailure"
         }
+     }
+     if [data][request_category] not in [ "files", "api", "oai2d", "static" ]  {
+        mutate { replace => {"[data][request_category]" => "ui"}}
+     }
+
+    {{ if .Values.logstash.cern_monit }}
+    mutate { rename => { "data" => "[m][data]" }}
+    mutate {
+      add_field => {"[m][producer]" => "zenodo"
+                    "[m][type]" => "zenodordm-access-logs"
+                    "[m][data][_environment]" => "rdm${ENVIRONMENT}"
+                 }
+      rename => {
+          "request_time" => "[m][data][request_time]"
+          "bytes_received" => "[m][data][bytes_received]"
+          "request_id" => "[m][data][request_id]"
+          "message" => "[m][data][message]"
+          "http_x_forwarded_for" => "[m][data][http_x_forwarded_for]"
+          "user_agent" => "[m][data][user_agent]"
+          "path" => "[m][data][path]"
+          "bytes" => "[m][data][bytes]"
+          "http_code" => "[m][data][http_code]"
+          "connect_time" => "[m][data][connect_time]"
+          "session_id" => "[m][data][session_id]"
+          "referer" => "[m][data][referer]"
+          "user_id" => "[m][data][user_id]"
+          "http_method" => "[m][data][http_method]"
+          "http_protocol" => "[m][data][http_protocol]"
+          "client_ip" => "[m][data][client_ip]"
+          "host" => "[m][data][host]"
+          "url" => "[m][data][url]"
+          "request_timestamp" => "[m][data][request_timestamp]"
+          "query_string" => "[m][data][query_string]"
+          "upstream_header_time" => "[m][data][upstream_header_time]"
+          "upstream_response_time" => "[m][data][upstream_response_time]"
       }
     }
+    mutate { rename => { "[m][data][country_name]" => "[m][data][country]" }}
```

Comment:
This is to be compatible with the legacy system, where the field was called `country`